sips/sip-033/sip-033-clarity4.md

```diff
@@ -0,0 +1,567 @@
+# Preamble
+
+SIP Number: 033
+
+Title: Clarity Smart Contract Language, version 4
+
+Author(s):
+
+- Brice Dobry <brice@stackslabs.com>
+- Marvin Janssen <marvin@ryder.id>
+- Jude Nelson <jude@stacks.org>
+- Rapha.btc <ra@fak.fun>
+- Adriano Di Luzio <adriano@bitcoinl2labs.com>
+
+Status: Draft
```

Comment:
Can you advance this to Activation-in-Progress @obycode?

Reply:
[c2e7310](https://github.com/stacksgov/sips/pull/218/commits/c2e7310ac34f30a8fa33b6805e705b2368a58cad)